c/tskit/trees.c

```diff
@@ -6549,7 +6635,7 @@ tsk_tree_seek_index(tsk_tree_t *self, tsk_id_t tree, tsk_flags_t options)
 }
 
 static int TSK_WARN_UNUSED
-tsk_tree_seek_linear(tsk_tree_t *self, double x, tsk_flags_t TSK_UNUSED(options))
```

Comment:
I don't understand why the unused flags are being removed? Won't ever be part of the public API?